cmd/loadbalancer/main.go

```diff
@@ -47,8 +47,8 @@ func main() {
 
 	// Register standard Go metric collectors, which are by default registered when using global registry.
 	reg.MustRegister(
-		version.NewCollector("observable-demo"),
-		prometheus.NewBuildInfoCollector(),
+		version.NewCollector(""),
+		prometheus.NewBuildInfoCollector(), // Why both? Can we choose one?
```

Comment:
ping @kakkoyun

Reply:
They include different extra information.
Version:

https://github.com/prometheus/common/blob/629b6ff9b3aafafba54ab96663903fde9544f037/version/info.go#L27-L34

Build Info:
https://github.com/prometheus/client_golang/blob/803ef2a759d7caaaa0de58e3815f1be4c8b5a42a/prometheus/go_collector.go#L388-L392

When I'm in doubt I choose both :D

If I had to choose one, I'd choose `version.NewCollector`